ocaml/util/xapi_version.ml

```diff
@@ -9,5 +9,9 @@
 let git_id = ""
 let hostname = "localhost"
 let date = Build_info.date
-let xapi_version_major = 1
-let xapi_version_minor = 20
+let (xapi_version_major, xapi_version_minor) =
+  try
+    Scanf.sscanf Build_info.version "%d.%d.%s" (fun maj min _rest -> (maj, min))
```

Comment:
I see that you rely on the tag having at least 3 components, for xapi that is probably a safe assumption.